### PR TITLE
chore: move README into docs/

### PR DESCRIPTION
**Pull request by lindsaylevine**

### CODE REVIEW

File: docs/custom-functions.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## Custom Netlify Functions
```
Comment (ascorbic) [inline docs/custom-functions.js:1]: Shouldn't this be a `.md` file?

File: docs/custom-redirects.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Custom Netlify Redirects
```
Comment (ascorbic) [inline docs/custom-redirects.js:1]: ...and this one

Comment (lindsaylevine) [reply docs/custom-redirects.js]: lol. yes.

**Comment by lindsaylevine** [comment]
fixed 🤡

**Review by ascorbic — APPROVED** [review]
We could probably have fewer of these pages, but I think this is good for now. I think we can find somewhere between "everything in the readme" and "a page per paragraph".

**Pull request merged**